ghost/core/core/server/data/migrations/utils/tables.js

```diff
@@ -38,7 +38,7 @@ function addTable(name, tableSpec) {
 /**
  * Creates migration which will drop a table
  *
- * @param {[string]} names  - names of the tables to drop
```

Comment:
`[string]` means that we accept an array with exactly one string in, whereas `string[]` is an array of zero or more strings